Assets/Mirage/Runtime/ClientObjectManager.cs

```diff
@@ -515,7 +510,7 @@ internal void OnRpcMessage(RpcMessage msg)
 
         void CheckForLocalPlayer(NetworkIdentity identity)
         {
-            if (identity && identity == LocalPlayer)
+            if (identity && identity == Client.Player.Identity)
```

Comment:
do we need to keep the null propagation? the previous use was `Client.Player?.Identity`

Reply:
fixed